test/hotspot/jtreg/runtime/appcds/cacheObject/ArchivedModuleWithCustomImageTest.java

```diff
@@ -110,7 +108,7 @@ private static void testArchivedModuleUsingImage(Path image)
         String[] dumpCmd = {
             customJava.toString(),
             "-XX:SharedArchiveFile=./ArchivedModuleWithCustomImageTest.jsa",
-            "-Xshare:dump"};
+            "-Xshare:dump", "-Xlog:cds"};
```

Comment:
Includes the changes from [JDK-8233826](https://bugs.openjdk.org/browse/JDK-8233826) ( commit [d1ad0ea](https://github.com/openjdk/jdk/commit/d1ad0eaf8fa69da267de5e4b6e1f1ecab48e8c73) ) on this file